contract/src/test/java/io/neow3j/contract/TransactionBuilderTest.java

```diff
@@ -663,56 +757,39 @@ public void invokeScriptWithoutSettingScript() throws IOException {
         Account sender = new Account(senderPair);
         Wallet w = Wallet.withAccounts(sender);
 
-        exceptionRule.expect(InvocationConfigurationException.class);
+        exceptionRule.expect(TransactionConfigurationException.class);
         exceptionRule.expectMessage("Cannot make an 'invokescript' call");
-        new Invocation.Builder(neow)
-                .withWallet(w)
-                .invokeScript();
-    }
-
-    @Test
-    public void invokeFunctionWithoutSettingContract() throws IOException {
-        Invocation.Builder b = new Invocation.Builder(neow)
-                .withFunction(NEP5_TRANSFER)
-                .withWallet(Wallet.createWallet());
-
-        exceptionRule.expect(InvocationConfigurationException.class);
-        exceptionRule.expectMessage("contract");
-        b.invokeFunction();
+        new TransactionBuilder(neow)
+                .wallet(w)
+                .callInvokeScript();
     }
 
     @Test
     public void invokeFunctionWithoutSettingFunction() throws IOException {
```

Comment:
This test method doesn't belong here anymore. It tests the behavior of SmartContract but not TransactionBuilder.

Reply:
True, actually it already exists in SmartContractTest, so I'll remove this test.